lib/Symbol.t

```diff
@@ -87,6 +90,7 @@ use Symbol qw(qualify qualify_to_ref);  # must import into this package too
 
 # tests for delete_package
 package main;
+no warnings 'once';
 $Transient::variable = 42;
```

Comment:
rather than disabling that warning in the global scope either
`$Transient::variable = $Transient::variable = 42;`
or 
`{ no warnings 'once'; $Transient::variable = 42; }`

Reply:
The test is complicated enough that I decided a more directed follow on fix would be a better choice for this.